projects/ngx-prx-styleguide/src/lib/datepicker/datepicker.component.spec.ts

```diff
@@ -33,15 +39,47 @@ describe('Component: DatepickerComponent', () => {
     expect(comp.picker instanceof Pikaday).toBe(true);
   });
 
-  it('emits dateChanged event when date is selected', () => {
-    spyOn(comp.dateChange, 'emit').and.stub();
+  it('should update date picker to reflect changes to @Input() date', () => {
+    expect(pikadaySetSpy).not.toBeCalled();
+    comp.date = new Date();
+    comp.date.setHours(0);
+    comp.date.setMinutes(0);
+    comp.date.setSeconds(0);
+    comp.date.setMilliseconds(0);
+    fix.detectChanges();
+    expect(pikadaySetSpy).toBeCalledTimes(1);
+    expect(pikadaySetSpy).toBeCalledWith(comp.date);
+
+    comp.date = new Date(comp.date.valueOf() - 24 * 60 * 60 * 1000);
+    fix.detectChanges();
+    expect(pikadaySetSpy).toBeCalledTimes(2);
+    expect(pikadaySetSpy).toBeCalledWith(comp.date);
+  });
+
+  it('should update date picker in UTC mode to reflect changes to @Input() date', () => {
+    expect(pikadaySetSpy).not.toBeCalled();
+    comp.UTC = true;
+    comp.date = new Date(Date.UTC(2018, 1, 1, 0, 0, 0, 0));
+    fix.detectChanges();
+    expect(pikadaySetSpy).toBeCalledTimes(1);
+    expect(pikadaySetSpy).toBeCalledWith(comp.pickerUTCOffset(comp.date));
+
+    comp.date = new Date(comp.date.valueOf() - 24 * 60 * 60 * 1000);
+    fix.detectChanges();
+    expect(pikadaySetSpy).toBeCalledTimes(2);
+    expect(pikadaySetSpy).toBeCalledWith(comp.pickerUTCOffset(comp.date));
+  });
+
+  // TODO: convert to integration test
```

Comment:
The rest of these tests can be re-written by mocking Pikaday functionality. This one exists only to test the integration between our datepicker component and Pikaday, and should move into an integration test suite.